feature-libs/organization/administration/core/services/organization-page-meta.resolver.ts

```diff
@@ -49,34 +53,38 @@ export class OrganizationPageMetaResolver
    */
   protected readonly ORGANIZATION_SEMANTIC_ROUTE = 'organization';
 
-  // TODO(#10467): Remove deprecated constructors
-  constructor(
-    contentPageMetaResolver: ContentPageMetaResolver,
-    translation: TranslationService,
-    semanticPath: SemanticPathService,
-    routingService: RoutingService,
-    // eslint-disable-next-line @typescript-eslint/unified-signatures
-    basePageMetaResolver?: BasePageMetaResolver
-  );
-  /**
-   * @deprecated since 3.1, we'll use the BasePageMetaResolver in future versions
-   */
-  constructor(
-    contentPageMetaResolver: ContentPageMetaResolver,
-    translation: TranslationService,
-    semanticPath: SemanticPathService,
-    routingService: RoutingService
-  );
   constructor(
     protected contentPageMetaResolver: ContentPageMetaResolver,
     protected translation: TranslationService,
     protected semanticPath: SemanticPathService,
-    protected routingService: RoutingService,
-    @Optional() protected basePageMetaResolver?: BasePageMetaResolver
```

Comment:
In migration docs it should be mentioned that this resolver no longer needs BasePageMetaResolver